<launch>
   <!-- Load the urdf into the parameter server. -->
   <param name = "robot_description" command = "xacro --inorder $(find robot_calibration_3d)/urdf/lrmate200ic.xacro" />
    
   <rosparam file = "$(find robot_calibration_3d)/config/3d_state_update_parameters.yaml" />

   <node pkg = "robot_state_publisher" type = "robot_state_publisher" name = "robot_state_publisher" />
   <node pkg = "robot_calibration_3d" type = "joint_state_publisher"  name = "joint_state_publisher_1" />
   <node type = "rviz" name = "rviz" pkg = "rviz" args = "-d $(find robot_calibration_3d)/rviz/simple_joint_state.rviz"/>

</launch>
